app/javascript/vue/navigation/notifications/notifications_flyout.vue

```diff
@@ -2,7 +2,9 @@
   <div class="sci--navigation--notificaitons-flyout">
     <div class="sci--navigation--notificaitons-flyout-title">
       {{ i18n.t('nav.notifications.title') }}
-      <i class="sn-icon sn-icon-close" @click="$emit('close')"></i>
+      <a class="ml-auto cursor-pointer" :href="'/users/settings/account/preferences'" :title="i18n.t('nav.settings')">
```

Comment:
This needs to be passed into the component, not hardcoded. I'm pretty sure it should already be available in `app/javascript/vue/navigation/top_menu.vue`, in the `userMenu` property, probably under the key `I18n.t('users.settings.sidebar.account_nav.preferences')`. Just pass that into the notification flyout component.